web/crux/src/app/health/health.dto.ts

```diff
@@ -0,0 +1,16 @@
+import { IsIn, IsOptional, IsString } from 'class-validator'
+
+export const SERVICE_STATUS_VALUES = ['unavailable', 'disrupted', 'operational'] as const
+export type ServiceStatusDto = (typeof SERVICE_STATUS_VALUES)[number]
+
+export class HealthDto {
+  @IsIn(SERVICE_STATUS_VALUES)
+  status: ServiceStatusDto
+
+  @IsString()
+  version: string
+
+  @IsString()
+  @IsOptional()
```

Comment:
Do not need the `@IsOptional()` if we are using `?`.

Reply:
I think it's necessary for the c# generator.

Reply:
We don't need.